charts/base/values.yaml

```diff
@@ -36,6 +36,11 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+ports:
+  - name: http
```

Comment:
why we have list/object for ports variable, it seems there is no usage of any sub-fields of this array/list in configs

Reply:
maybe we need another name for "ports" variable(for example portsEnabled) to not confuse with existing "ports" field in deployment.yaml ?